src/github/pullRequestOverview.ts

```diff
@@ -702,7 +702,7 @@ export class PullRequestOverviewPanel {
 			<html lang="en">
 			<head>
 				<meta charset="UTF-8">
-				<meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src vscode-resource: https:; script-src 'unsafe-eval' 'nonce-${nonce}'; style-src vscode-resource: 'unsafe-inline' http: https: data:;">
```

Comment:
Ohhh I bet this is the change that caused that `EvalError` you were seeing yesterday ☝️ 

It's a very good change to make, though.

Reply:
Heh yup, and totally!